chapter1.md

```diff
@@ -4,13 +4,11 @@ Elita Baldridge, *elita.baldridge@weecology.org, Utah State University*, David J
 
 # Introduction
 
-The species abundance distribution (SAD) describes the full distribution of commonness and rarity in ecological systems. It is one of the most fundamental and ubiquitous patterns in ecology, and exhibits a consistent general form with many rare species and few abundant species occurring within a community. This general shape is often referred to as a hollow curve.
+The species abundance distribution (SAD) describes the full distribution of commonness and rarity in ecological systems. It is one of the most fundamental and ubiquitous patterns in ecology, and exhibits a consistent general form with many rare species and few abundant species occurring within a community. The SAD is one of the most widely studied patterns in ecology, leading to a proliferation of models that attempt to characterize the shape of the distribution and identify potential mechanisms for the pattern [see @mcgill2007 for a recent review of SADs]. These models range from arbitrary distributions that are chosen based on providing a good fit to the data [@fisher1943], to distributions chosen based on the most likely states of generic random systems [@frank2011; @harte2011; @locey2013], to models based more directly on ecological processes [@tokeshi1993; @hubbell2001; @volkov2003; @alroy2015].
 
-The SAD is one of the most widely studied patterns in ecology, leading to a proliferation of models that attempt to characterize the shape of the distribution and identify potential mechanisms for the pattern [see @mcgill2007 for a recent review of SADs]. These models range from arbitrary distributions that are chosen based on providing a good fit to the data [@fisher1943], to distributions chosen based on the most likely states of generic random systems [@frank2011; @harte2011; @locey2013], to models based more directly on ecological processes [@tokeshi1993; @hubbell2001; @volkov2003; @alroy2015].
+Which model or models provide the best fit to the data, and the resulting implications for the processes structuring ecological systems, is an active area of research [e.g., @mcgill2003; @volkov2003; @ulrich2010; @white2012; @connolly2014]. However, most comparisons of the different models: 1) use only a small subset of available models [typically two; e.g., @mcgill2003; @volkov2003; @white2012; @connolly2014]; 2) focus on a single ecosystem or taxonomic group [e.g., @mcgill2003; @volkov2003]; or 3) fail to use the most appropriate statistical methods [e.g., @ulrich2010]. This makes it difficult to draw general conclusions about which, if any, models provide the best empirical fit to species abundance distributions.
```

Comment:
Should we cite the "best practices" paper here, so it's clearer what's wrong with @ulrich2010?